dygie/tests/data/annotated_doc_test.py

```diff
@@ -0,0 +1,259 @@
+       
+        # Set up entities 
+        self.ents = [ad.Ent("T4\tPerson 59 62\tShe".split()),
+                     ad.Ent("T5\tPersonnel.Election 67 74\telected".split()),
+                     ad.Ent("T6\tYear 78 82\t2017".split())]
+        ad.Ent.char_to_token(self.ents, self.sents, self.nlp)
+
+        # Set up events 
+        self.event1 = ad.Event("E1\tPersonnel.Election:T5 Person:T4 Year:T6".split())
+
+        # Right answer 
+        self.events = [[],[],
+                [[[16, "Personnel.Election"],[14, 14, "Person"],[18, 18, "Year"]]]]
+
+
+    def test_set_arg_objects(self):
+        
+        self.event1.set_arg_objects(self.ents)
+
+        self.assertEqual(self.event1.trigger, self.ents[1])
+        self.assertEqual(self.event1.args, [self.ents[0], self.ents[2]])
+
+
+    def test_format_events_dygiepp(self):
+
+        self.event1.set_arg_objects(self.ents)
+        events = ad.Event.format_events_dygiepp([self.event1], self.sent_idx_tups)
+
+        self.assertEqual(events, self.events)
+
+
+class TestEquivRel(unittest.TestCase):
+
+    def setUp(self):
+
+        # Set up document text 
+        self.nlp = spacy.load("en_core_web_sm") 
+        self.text = ("Seattle is a rainy city. Jenny Durkan is the city's mayor. "
+                    "She was elected in 2017.")
+        self.sents = [[tok.text for tok in sent] for sent in self.nlp(self.text).sents]
+        self.sent_idx_tups = [(0, 6), (6, 14), (14, 19)]
+        # NOTE: spacy tokenizes words with apostrophes into separate words.
+
+        # Set up entities 
+        self.ents = [ad.Ent("T1\tCity 0 7\tSeattle".split()),
+                     ad.Ent("T2\tPerson 25 37\tJenny Durkan".split()),
+                     ad.Ent("T3\tCity 41 51\tthe city's".split()),
+                     ad.Ent("T4\tPerson 59 62\tShe".split()),
+                     ad.Ent("T5\tPersonnel.Election 67 74\telected".split()),
+                     ad.Ent("T6\tYear 78 82\t2017".split())]
+        ad.Ent.char_to_token(self.ents, self.sents, self.nlp)
+
+        # Set up equivalence relations 
+        self.equivrel1 = ad.EquivRel("*\tEQUIV T1 T3".split())
+        self.equivrel2 = ad.EquivRel("*\tEQUIV T2 T4".split())
+
+        # The dygiepp-formatted correct answer 
+        self.corefs = [[[0,0],[9,11]],[[6,7],[14,14]]]
+
+
+    def test_set_arg_objects(self):
+
+        self.equivrel1.set_arg_objects(self.ents)
+        self.equivrel2.set_arg_objects(self.ents)
+
+        self.assertEqual(self.equivrel1.args, [self.ents[0], self.ents[2]])
+        self.assertEqual(self.equivrel2.args, [self.ents[1], self.ents[3]])
+
+
+    def test_format_corefs_dygiepp(self):
+
+        self.equivrel1.set_arg_objects(self.ents)
+        self.equivrel2.set_arg_objects(self.ents)
+        corefs = ad.EquivRel.format_corefs_dygiepp([self.equivrel1, self.equivrel2])
+
+        self.assertEqual(corefs, self.corefs)
+
+
+class TestAnnotatedDoc:
```

Comment:
Should subclass `unittest.TestCase`; otherwise it doesn't run when invoked using `python -m unittest annotated_doc_test.py`.

Reply:
This test broke when run, with error 

```
======================================================================
ERROR: test_format_dygiepp (annotated_doc_test.TestAnnotatedDoc)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/dwadden/proj/dygiepp/dygie/tests/data/annotated_doc_test.py", line 250, in test_format_dygiepp
    annotated_doc = ad.AnnotatedDoc.parse_ann(self.txt, self.ann, self.nlp, self.dataset)
TypeError: parse_ann() missing 1 required positional argument: 'coref'
```